app/services/csv_generator.rb

```diff
@@ -22,10 +22,7 @@ def attributes
   private
 
   def row(record)
-    record
-      .attributes
-      .slice(*attributes)
-      .map { |key, value| format(key, value) }
+    attributes.map { |attribute| format(attribute, record.send(attribute)) }
```

Comment:
You may well be able to `record[attribute]` here instead of `send`. If not, can we use `public_send` instead?